<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeWindow</class>
 <widget class="QMainWindow" name="TreeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>H&amp;S Debug</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>536</width>
      <height>210</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>Node</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Depth</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Count</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHS_Debug">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSave_As"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionChosen_Action"/>
    <addaction name="actionShow_Parameters"/>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
   </widget>
   <widget class="QMenu" name="menuExpand">
    <property name="title">
     <string>Expand</string>
    </property>
    <addaction name="actionExpand_Until_This"/>
    <addaction name="actionExpand_All"/>
    <addaction name="actionCollapse_All"/>
   </widget>
   <addaction name="menuHS_Debug"/>
   <addaction name="menuExpand"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExpand_Until_This">
   <property name="text">
    <string>Expand Until This</string>
   </property>
  </action>
  <action name="actionChosen_Action">
   <property name="text">
    <string>Extra Statistics</string>
   </property>
  </action>
  <action name="actionShow_Parameters">
   <property name="text">
    <string>Show Parameters</string>
   </property>
  </action>
  <action name="actionExpand_All">
   <property name="text">
    <string>Expand All</string>
   </property>
  </action>
  <action name="actionCollapse_All">
   <property name="text">
    <string>Collapse All</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
